examples/app_blink_k20_isr.rs

```diff
@@ -51,3 +46,9 @@ pub fn main() {
       wfi();
   }
 }
+
+#[start]
+fn start(_: isize, _: *const *const u8) -> isize {
```

Comment:
Happy to 'keep it simple' and have this (because I know how annoying it is defining `#start` in Zinc). But for 1.0 we should really have pre-defined `#start` and it should be calling `init_stack` and `init_data`.